### ISSUE TITLE
Artwork seems to be lost on convert

### ISSUE THREAD

**Issue by ziggythehamster** [reupen/ipod_manager#12]
I've got a lot of FLACs, and when they're converted to AAC for my iPod using Nero AAC, it seems like the artwork tags get lost, and it seems like the foo_dop process of figuring out what the artwork is supposed to be doesn't work because it starts operating on the converted file for info rather than the original. I'm using the default command line for Nero AAC, which doesn't do anything special with the tagger.

**Comment by reupen** [comment]
Hi,

Could you confirm:

- how the artwork settings are configured on the iPod Features tab in iPod manager preferences
- if this happens straight after adding a new file
- the version of foobar2000 you're using

Thanks

**Comment by ziggythehamster** [comment]
So I started typing up a response with screenshots, and to my surprise, my iPod actually did have artwork this go around. The only things that are different are that I've rebooted my computer now and I reformatted my iPod because the SD card I was using was not working correctly.

So I guess, this is actually working correctly. I tried .mp3, .flac, and .vgz, and all have artwork either coming from the cover file or the embedded artwork. My mistake!
